monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -156,67 +112,25 @@ def export_to_disk(self, output_path: str, algo_name: str, **kwargs):
-        self.cfg["bundle_root"] = write_path
-        os.makedirs(write_path, exist_ok=True)
-        # handling scripts files
-        output_scripts_path = os.path.join(write_path, "scripts")
-        if os.path.exists(output_scripts_path):
-            shutil.rmtree(output_scripts_path)
-        if self.scripts_path is not None and os.path.exists(self.scripts_path):
-            shutil.copytree(self.scripts_path, output_scripts_path)
-        # handling config files
-        output_config_path = os.path.join(write_path, "configs")
-        if os.path.exists(output_config_path):
-            shutil.rmtree(output_config_path)
-
-        # break the config into multiple files and save
-        sections = ["network", "transforms_infer", "transforms_train", "transforms_validate"]
-        self.save_config_files(output_config_path, sections)
-        logger.info(write_path)
-        self.output_path = write_path
-
-    def save_config_files(self, output_config_path, sections):
-        """
-        Save the auto-generated config files into multiple files.
-
-        Args:
-            output_config_path: path to save the files
-            sections: the sections that will be picked up and individually saved.
+            kwargs: other parameters, including: "copy_dirs=True/False" means whether to copy the template as output
+                instead of inplace operation, "fill_template=True/False" means whether to fill the placeholders
+                in the template. other parameters are for `fill_template_config` function.
 
         """
-        os.makedirs(output_config_path, exist_ok=True)
-
-        config_body = self.cfg.get()
-        for s in sections:
-            config_section_file = os.path.join(output_config_path, s + ".yaml")
-            config_section = _pop_args(config_body, s)
-            section_body = {}
-            section_body[s] = config_section[0]  # _pop_args returns a Tuple even if there is only one element.
-            ConfigParser.export_config_file(section_body, config_section_file, fmt="yaml", default_flow_style=None)
-
-        output_config_file = os.path.join(output_config_path, "hyper_parameters.yaml")
-        ConfigParser.export_config_file(config_body, output_config_file, fmt="yaml", default_flow_style=None)
-
-        with open(output_config_file, "r+") as f:
-            lines = f.readlines()
-            f.seek(0)
-            f.write(f"# Generated automatically by `{__name__}`\n")
-            f.write("# For more information please visit: https://docs.monai.io/\n\n")
-            for item in ensure_tuple(self.template_configs):
-                f.write(f"# source file: {item}\n")
-            f.write("\n\n")
-            f.writelines(lines)
+        if kwargs.pop("copy_dirs", True):
+            self.output_path = os.path.join(output_path, algo_name)
+            os.makedirs(self.output_path, exist_ok=True)
+            shutil.copytree(self.template_path, self.output_path)
```

Comment:
copytree will throw an error if the self.output_path exists. I would recommend deleting it or moving the folder to a backup location:

```python
if os.path.isdir(self.output_path):
    shutil.rmtree(self.output_path)
shutil.copytree(self.template_path, self.output_path)
```

Reply:
updated in the latest commit.

Thanks.